GNUmakefile.osx

```diff
@@ -102,6 +102,7 @@ endif
 
 ifndef LIBXML2_LIBS
     LIBXML2_LIBS := $(shell xml2-config --libs)
+    LIBXML2_LIBS := $(filter-out -L$(shell xcrun --show-sdk-path)/usr/lib, $(LIBXML2_LIBS))
```

Comment:
Can you explain this line please?  I don't even have the xcrun command on my Fedora installation.  What is this supposed to do?